tutor/static/css/django_private_chat.css

```diff
@@ -70,17 +70,20 @@
     font-weight: normal;
 }
 
-.online-status {
+.text-success {
```

Comment:
Won't this overwrite the bootstrap text-success class? http://getbootstrap.com/css/#helper-classes-colors